src/judging_round_display.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+
+
+class JudgingRoundDisplay extends React.Component {
+    state = {
+        data: {},
+    }
+
+    componentDidUpdate(prevProps) {
+	const judging_round_id = this.props.judging_round
+	if (judging_round_id &&
+	      prevProps.judging_round !== this.props.judging_round) {
+	    this.fetch_data()
+	}
+    }
+
+    fetch_data() {
+	fetch("http://localhost:8000/api/v1/judging_round/" +
```

Comment:
ES6 introduces template strings which gives the ability to interpolate values. You make them using backticks instead of quotes so you could write this as:

```js
fetch(`http://localhost:8000/api/v1/judging_round/${judging_round_id}/`
```

Reply:
Addressed in AC-5696.